src/system/BoxWithHeader/index.js

```diff
@@ -0,0 +1,6 @@
+/**
```

Comment:
Can you create a typescript file? Also, since this is a simple component, you don't need to import from a index file, you can import directly from the file: `export const BoxWithHeader`.